load_game.py

```diff
@@ -357,45 +368,45 @@ def load_post_blitz():
 
     while not screen.Minimap1():
         if screen.Minimap4():
-            FFXC.set_value("axis_lx", -1)
-            FFXC.set_value("axis_ly", -1)
+            FFXC.set_value("AxisLx", -1)
```

Comment:
I'd check these too, but my money says the shouldn't be changed either.

Reply:
Best is to have @shenef check I think.

Reply:
Can blanket fix these too. Hang tight.